### DISCUSSION TITLE
Disable zoom on TextBox focus on WASM mobile

### DISCUSSION THREAD

**Discussion by rullinoiz** [AvaloniaUI/Avalonia#17417] [Q&A]
How do I stop it from zooming in on TextBoxes when they are focused? It's impossible to zoom back out and it makes it impossible to navigate the app in this state
![IMG_8135](https://github.com/user-attachments/assets/affb0725-8ceb-4fdf-9c6a-0a3a684d563c)

**Comment by fraluderin** [comment]
I have the same issue when running in Safari and Edge on iOS. I also tried to apply this to my MainView, but it didn't work.

```
<UserControl.GestureRecognizers>
	<PinchGestureRecognizer />
</UserControl.GestureRecognizers>
```

Unfortunately this issue makes web pages totally unusable on mobile browsers.

**Comment by Gillibald** [comment]
Mobile wasm isn't supported and most likely never will be. Just target mobile directly.

**Reply by rullinoiz** [reply]
well that's lame. there isn't even a way to make it so that mobile gestures aren't ignored? the way i'm deploying this would make it very time consuming if i were to have to target mobile directly (i have no money for the app store fees)

**Comment by Gillibald** [comment]
The browser backend supports tracking the safe area. You could try to work with that.

https://github.com/AvaloniaUI/Avalonia/blob/01a7c859d4c4d46a23eb4c152217c106cd9318bb/samples/ControlCatalog/MainView.xaml.cs#L101
